### PR TITLE
DEV-1 + API-369 updates

### PR DESCRIPTION
**Pull request by bc-mkatz**
The updates to *store_information* are for DEV-1, and should be deployed circa 10/19/16. The *page/s* files are new drafts for API-369 – rough placeholders, but safe to merge because they’re not linked anywhere. Other files got minor copy edits.

### CODE REVIEW

File: source/includes/_api_CRUD_pages.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+*   OAuth
+ `GET /stores/{store_hash}/v2/pages/`
+*   Basic Auth
+ `GET /api/v2/pages`
+
+#### <span class="jumptarget"> Filters </span>
+
+Filter parameters can be added to the URL query string to select specific pages in the collection.
+
+**[Lawrence: Need filters. The struck-out text below is copied/pasted from the Blog Posts API docs. Thanks.]**
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_pages.md]: This shouldn't go into live copy

File: source/includes/_api_CRUD_pages.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+| --- | --- | --- |
+| is_published | string | /stores/{store_hash}/v2/blog/posts?is_published={value} |
+| url | string | /stores/{store_hash}/v2/blog/posts?url={value} |
+</strike>
+
+#### <span class="jumptarget"> Pagination </span>
+
+Parameters can be added to the URL query string to paginate the collection. The maximum limit is 250. If a limit isn’t provided, up to 50 pages are returned by default.
+
+**[Lawrence: Need paginators. The struck-out text below is copied/pasted from the Blog Posts API docs. Thanks.]**
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_pages.md]: Also needs removal

File: source/includes/_api_CRUD_pages.md [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+### <span class="jumptarget"> Create a Page </span>
+
+Creates a new page.
+
+*   OAuth
+`POST /stores/{store_hash}/v2/pages`
+*   Basic Auth
+`POST /api/v2/pages`
+
+**[Lawrence: Sorry, but I need everything for this one – any read-only properties, any required properties, and any usage notes; plus, a sample Request body and (ideally) a sample JSON Response. Struck-out text below is copied/pasted from Blog Posts API. Thanks.]**
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_pages.md]: Needs removal

File: source/includes/_api_CRUD_store_information.md [hunk 4]
```diff
@@ -23,10 +23,24 @@ Example JSON returned in the response:
...
+  "country": "United States",
   "phone": "567-098-9274",
   "admin_email": "admin@bigcommerce.com",
   "order_email": "order@bigcommerce.com",
+  "timezone": {
+    "name": "America/Vancouver",
+    "raw_offset": -28800,
+    "dst_offset": -25200,
+    "dst_correction": true,
+    "date_format": {
```
Comment (bc-AlyssNoland) [inline source/includes/_api_CRUD_store_information.md]: Do we have annotation in the Store Information Reference for [PHP date](http://php.net/manual/en/function.date.php) formatting?

File: source/includes/_api_objects_page.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+| mobile_body | text | HTML or variable that populates this page's `<body>` element, in mobile view. |
+| has_mobile_version | boolean | If `true`, this page has a mobile version. |
+| is_visible | boolean | If `true`, this page appears in the storefront's navigation menu. (??) |
+| is_homepage | boolean | If `true`, this page is the storefront's home page. |
+| meta_title | string | Text specified for this page's `<title>` element. (If empty, the value of the `name` property is used.) |
+| layout_file | string | Layout template for this page. |
+| sort_order | int | Order in which this page should display on the storefront. (??) Lower integers give the page higher prominence. |
+| meta_title | string | Text contained within this page's `<title>` element. (??) |
+| search_keywords | string | Comma-separated list of keywords that can be used to locate this page when searching the store. (??) |
+| meta_keywords | string | Comma-separated list of meta keywords to include in the HTML. (?? Should this be rewritten as: "Comma-separated list of SEO-relevant keywords to include in the page's `<meta>` element." ??) |
```
Comment (bc-AlyssNoland) [inline source/includes/_api_objects_page.md]: We should remove the things sandwiched with question marks.

File: source/includes/_api_objects_store_information.md [hunk 6]
```diff
@@ -10,11 +10,14 @@ A Store Information object represents an individual store's profile and metadata
...
-| name | string | The store's name. |
+| secure_URL | string | Store's current HTTPS URL. |
+| name | string | Store's name. |
+| first_name | string | Primary contact's first name (as defined during the store sign-up process). |
+| last_name | string | Primary contact's last name (as defined during the store sign-up process). |
 | address | string | Display address. |
+| country | string | Country where the store is located (as defined during the store sign-up process). |
 | phone | string | Display phone number. |
-| admin_email | string | Email address of the store administrator. |
+| admin_email | string | Email address of the store administrator (primary contact). |
```
Comment (bc-AlyssNoland) [inline source/includes/_api_objects_store_information.md]: Primary contact is internal BC language. Might want to change to store admin/store owner or something like that.

File: source/includes/_api_CRUD_pages.md (hunk 1 above)
Comment (bc-mkatz) [inline source/includes/_api_CRUD_pages.md]: It won't. It's not linked anywhere. It got bundled into this commit. Lawrence W. is helping me finish it, before I link it anywhere.

File: source/includes/_api_CRUD_pages.md (hunk 3 above)
Comment (bc-mkatz) [inline source/includes/_api_CRUD_pages.md]: Same.

File: source/includes/_api_CRUD_store_information.md (hunk 4 above)
Comment (bc-mkatz) [inline source/includes/_api_CRUD_store_information.md]: Not that I've heard of. This date object was added to the API (and docs) by Jeff Hubbard, several weeks ago.

File: source/includes/_api_objects_store_information.md [hunk 7]
```diff
@@ -10,11 +10,14 @@ A Store Information object represents an individual store's profile and metadata
...
 | timezone | object | An object that defines the store's time zone, following conventions of the PHP `date` function. (For&#160;details, please see: <a href="http://php.net/manual/en/function.date.php" target="_blank">http://php.net/manual/en/function.date.php</a>.) This object contains the following elements:<br> `name`: a string identifying the time zone, in the format: <NOBR>`<Continent-name>/<City-name>`.</nobr><br> `raw_offset`: a negative or positive number, identifying the offset from UTC/GMT, in seconds, during winter/standard time.<br> `dst_offset`: -/+ (number) offset from UTC/GMT, in seconds, during summer/daylight saving time.<br> `dst_correction`: a boolean indicating whether this time zone observes daylight saving time.<br> `date_format` is a nested object, which contains the following internal elements:<br> * `display`: string that defines dates' display format, in the pattern: <NOBR>`M jS Y`;</nobr> <br> * `export`: string that defines the CSV export format for orders, customers, and products, in the pattern: <NOBR>`M jS Y`;</nobr> <br> * `extended_display`: string that defines dates' extended-display format, in the pattern: <NOBR>`M jS Y @ g:i A`.</nobr> |
```
Comment (bc-mkatz) [inline source/includes/_api_objects_store_information.md]: Excellent.

File: source/includes/_api_objects_store_information.md (hunk 6 above)
Comment (bc-mkatz) [inline source/includes/_api_objects_store_information.md]: Thanks for catching this. Just pushed that correction.

**Pull request merged**